plugins/ag-charts-generate-chart-thumbnail/src/executors-utils.ts

```diff
@@ -97,6 +99,83 @@ export function batchExecutor<ExecutorOptions>(
     };
 }
 
+const workers: ChildProcess[] = [];
+const terminateWorkers = () => {
+    workers.forEach((w) => w.kill('SIGKILL'));
+    workers.length = 0;
+};
+
+process.on('exit', () => {
+    terminateWorkers();
+});
+
+export function batchWorkerExecutor<ExecutorOptions>(worker: () => ChildProcess) {
+    const maxChildCount = Math.max(1, os.cpus().length - 1);
+    const results: Record<string, Promise<BatchExecutorTaskResult>> = {};
+    const resolvers: Record<string, (res: BatchExecutorTaskResult) => void> = {};
```

Comment:
Minor - but a `Map` would be better here

Reply:
Fixed